sap-ha-support/xml/art-sap-ha-support.xml

```diff
@@ -12,10 +12,31 @@
          version="5.0"
          xml:lang="en"
          xmlns:xi="http://www.w3.org/2001/XInclude"
-         xmlns:xlink="http://www.w3.org/1999/xlink">
+         xmlns:xlink="http://www.w3.org/1999/xlink"
+         xmlns:its="http://www.w3.org/2005/11/its">
   <title>Supported &ha; Solutions by &productname;</title>
   <info>
    <date><?dbtimestamp format="B d, Y" ?></date>
+   <!-- History -->
+   <revhistory xml:id="rh-article-sap-ha-support">
+    <revision><date>2023-03-09</date>
+      <revdescription>
+        <para>
+          Initial published
+        </para>
+      </revdescription>
+     </revision>
+    </revhistory>
+
+   <!-- Series -->
+    <meta name="series" its:translate="no">Products &amp; Solutions</meta>
+
+   <!-- Task -->
+    <meta name="task" its:translate="yes">
```

Comment:
Maybe my brain isn't fully working today after vacation, but in my humble opinion the `its:translate` attribute is wrong. Shouldn't it be "no"?

```suggestion
    <meta name="task" its:translate="no">
```

Otherwise it would be inconsistent with series.

Reply:
This is no bug, it's a feature. Therefore it stays exactly as it is.